assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -51,10 +51,12 @@ function useUploadVideoFrame({ updateMediaElement }) {
         id: posterId,
         source_url: poster,
         media_details: { width: posterWidth, height: posterHeight },
-      } = await uploadFile(obj);
+      } = await uploadFile(obj, {
+        post: id,
+        media_source: 'poster-generation',
+      });
       // Meta data cannot be sent as part of upload.
       await updateMedia(posterId, {
```

Comment:
Going forward, we should remove this api call, as it will speed up posters being displayed.

Reply:
#5640 should remove it, no?

Reply:
Yeah, sorry, I should have linked to that PR.